lib/hotel/widgets/hotel_card.dart

```diff
@@ -0,0 +1,122 @@
+import 'package:flutter/material.dart';
+import 'package:overnites/app/widgets/text.dart';
+import 'package:overnites/hotel/model.dart';
+
+class HotelCard extends StatefulWidget {
+  const HotelCard({super.key, required this.hotel});
+
+  final Hotel hotel;
+
+  @override
+  State<HotelCard> createState() => _HotelCardState();
+}
+
+class _HotelCardState extends State<HotelCard> {
+  late final Hotel hotel;
+
+  @override
+  void initState() {
+    hotel = widget.hotel;
+    super.initState();
+  }
+
+  @override
+  Widget build(BuildContext context) {
+    return DecoratedBox(
+      decoration: BoxDecoration(
+        borderRadius: const BorderRadius.all(
+          Radius.circular(25),
+        ),
+        color: Theme.of(context).colorScheme.surface,
+      ),
+      child: Flex(
+        direction: Axis.vertical,
+        children: [
+          SizedBox.fromSize(
+            size: const Size.fromHeight(175),
+            child: const DecoratedBox(
+              decoration: BoxDecoration(
+                color: Color(0xFFD9D9D9),
+                borderRadius: BorderRadius.all(
+                  Radius.circular(25),
+                ),
+              ),
+            ),
+          ),
+          Padding(
+            padding: const EdgeInsets.fromLTRB(20, 8, 20, 20),
+            child: Flex(
+              direction: Axis.vertical,
+              crossAxisAlignment: CrossAxisAlignment.start,
+              children: [
+                Text(
+                  hotel.name,
+                  style: TextStyle(
+                    fontSize: 32,
+                    color: Theme.of(context).colorScheme.onPrimary,
+                    fontWeight: FontWeight.bold,
+                  ),
+                ),
+                SizedBox.fromSize(
+                  size: const Size.fromHeight(4),
+                ),
+                TextWidget(
```

Comment:
Doesn't handle long descriptions that good

Reply:
Fixed